main_test.go

```diff
@@ -995,7 +997,9 @@ func TestRotate(t *testing.T) {
 				macFinUsers := map[string]struct{}{}
 				for _, row := range tc.MACFinIn {
 					if row.Username != "" {
-						macFinUsers[row.Username] = struct{}{}
+						if _, ok := macFinUsers[strings.ToLower(row.Username)]; !ok {
```

Comment:
why this change?

Reply:
This change is needed because the code now removes duplicates from the MACFin sheet, and then sets usernames to lowercase.
If I revert this change then the test on main_test.go:1031 fails because `Users are now missing from the MACFin sheet`.

Reply:
ah, so you need the lowercase, but I still don't see why you need the `if`

Reply:
Yup, you are correct. I don't need the `if` statement. Done.